atbuild/src/main.swift

```diff
@@ -65,10 +65,7 @@ if Process.arguments.contains("--help") {
     exit(1)
 }
 
-let package = loadPackageFile()
-print("Building package \(package.name)...")
-
-func runtask(taskName: String) {
+func runtask(taskName: String, package: Package) {
```

Comment:
Maybe we should change this to: `func runTask(task: Task)` and put the task lookup outside of this function?